src/field/FieldDriver.hh

```diff
@@ -258,8 +258,9 @@ CELER_FUNCTION DriverResult FieldDriver<StepperT>::accurate_advance(
         }
         else
         {
-            h = max(max(output.proposed_step, shared_.minimum_step),
```

Comment:
Is max (max ...) still replaced by clamp ?

Reply:
No, this is the result of two transformations:
- your fix: `max(max(` -> `min(max(`
- replace min/max with clamp

Reply:
Haa, so many reasons to retire!  Thanks!!1